pkg/util/pravegacluster.go

```diff
@@ -108,3 +115,48 @@ func Min(x, y int32) int32 {
 	}
 	return x
 }
+
+func ListSubTreeBFS(conn *zk.Conn, root string) (*list.List, error) {
```

Comment:
This function should be moved to some other place. Maybe you can create a `zookeeper_util.go` file.

Reply:
Sounds good